R/utilities-sensitivity-analysis.R

```diff
@@ -77,8 +77,23 @@ individualSensitivityAnalysis <- function(simFilePath,
                                           resultsFileName = resultsFileName) {
   # Load simulation to determine number of parameters valid for sensitivity analysis
   sim <- loadSimulation(simFilePath)
+
+  allVariableParameterPaths <- ospsuite::potentialVariableParameterPathsFor(simulation = sim)
+
+
   if (is.null(variableParameterPaths)) { # If no parameters to vary specified, vary all parameters valid for sensitivity analysis
-    variableParameterPaths <- ospsuite::potentialVariableParameterPathsFor(simulation = sim)
+    variableParameterPaths <- allVariableParameterPaths
+  } else {
+    #if a variableParameterPaths input is provided, ensure that all its elements exist within allVariableParameterPaths.  If not, give an error.
+
+    if(!all( variableParameterPaths %in% allVariableParameterPaths) ){
+      msg<- paste("Variable parameter path(s)",

```

Comment:
Let's define a single file with all the error messages as we have done with other libs. That way, if there is a typo or the message needs to change, we only look at the one place

Reply:
see issue https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/issues/27

Reply:
Changed. I used validateIsIncluded.   Should we add logError commands to all error checks? @pchelle

Reply:
Would it make sense to defien your own stop method that basiaclly logError and then stops?

Reply:
Added this to error-checks.R